packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,138 @@
+import { css } from "@microsoft/fast-element";
+import { display, focusVisible } from "../../styles";
+import {
+    neutralDividerRestBehavior,
+    neutralFocusBehavior,
+    neutralForegroundActiveBehavior,
+    neutralForegroundFocusBehavior,
+    neutralForegroundRestBehavior,
+} from "../../styles/recipes";
+import { heightNumber } from "../../styles/size";
+import { forcedColorsStylesheetBehavior } from "../../styles/match-media-stylesheet-behavior";
+import { SystemColors } from "../../styles/system-colors";
+
+export const AccordionItemStyles = css`
+    ${display("flex")} :host {
+        box-sizing: border-box;
+        font-family: var(--body-font);
+        flex-direction: column;
+        font-size: var(--type-ramp-minus-1-font-size);
+        font-weight: 400;
```

Comment:
Do you need to set font-weight?

Reply:
probably not, I think I was just following patterns elsewhere